pkg/ovs/openflow/interfaces.go

```diff
@@ -373,6 +373,7 @@ type PacketOutBuilder interface {
 	SetOutport(outport uint32) PacketOutBuilder
 	AddLoadAction(name string, data uint64, rng *Range) PacketOutBuilder
 	AddLoadRegMark(mark *RegMark) PacketOutBuilder
+	AddResubmitAction(inPort *uint16, table *uint8) PacketOutBuilder
```

Comment:
Just want to double confirm: why here you want "resubmit" but not "goto_table"? Do you plan to resubmit the packet to a previous table, or resubmit the packet to another in_port number?

Reply:
PacketOut can carry a list of OFActions. `Resubmit` is an action, but `goto_table` is an instruction instead of an action.